main/admin/resume_session.php

```diff
@@ -217,6 +217,76 @@
 </tr>
 
 <?php
+$sessionField = new SessionField();
+$sessionFields = $sessionField->get_all();
+

```

Comment:
You can create a function in the parent class something like this:
<pre>
echo $sessionField->getAllData($sessionId);
</pre>

That function will do this loop, the other extra fields (course, user, etc) can use the same loop